services/classic/management/virtualmachine/client.go

```diff
@@ -89,7 +311,33 @@ func (vm VirtualMachineClient) GetDeploymentName(cloudServiceName string) (strin
 	if cloudServiceName == "" {
 		return "", fmt.Errorf(errParamNotSpecified, "cloudServiceName")
 	}
-	requestURL := fmt.Sprintf(azureListDeploymentsInSlotURL, cloudServiceName)
+	requestURL := fmt.Sprintf(azureDeploymentSlotURL, cloudServiceName, "Production")
```

Comment:
Shouldn't the third param be `DeploymentSlotProduction`?

Reply:
Or perhaps `GetDeploymentName()` calls `GetDeploymentNameForSlot()`?

Reply:
good suggestions. I'll make the changes